arch/csr/Ssqosid/srmcfg.yaml

```diff
@@ -0,0 +1,110 @@
+# yaml-language-server: $schema=../../../schemas/csr_schema.json
+$schema: csr_schema.json#
+kind: csr
+name: srmcfg
+long_name: Supervisor Resource Management Configuration
+address: 0x181
+priv_mode: S
+length: SXLEN
+definedBy: Ssqosid
+description:
+  - id: csr-srmcfg-purpose
+    normative: true
+    text: |
+      The srmcfg register is used to configure a Resource Control ID (RCID) and a Monitoring Counter ID (MCID).
+      Both RCID and MCID are WARL fields.
+
+  - id: csr-srmcfg-field-usage
+    normative: true
+    text: |
+      The RCID and MCID accompany each request made by the hart to shared resource controllers.
+      The RCID is used to determine the resource allocations (e.g., cache occupancy limits, memory bandwidth limits, etc.) to enforce.
+      The MCID is used to identify a counter to monitor resource usage.
+
+  - id: csr-srmcfg-default-scope
+    normative: true
+    text: |
+      The RCID and MCID configured in the srmcfg CSR apply to all privilege modes of software execution on that hart by default,
```

Comment:
add backticks around `srmcfg` and also in the rest of the description, along with `mstateen0`.